### PR TITLE
[TASK] Use snabbdom for DOM

### PR DESCRIPTION
**Pull request by xf-**
## Description

Replaces virtual-dom

## Motivation and Context

Virtual-dom is unmaintained and slower. I compared different library and this was fest solution with patch functionality

## How Has This Been Tested?

Browserstack

## Checklist:

- [x] My code follows the code style of this project. (CI will test it anyway and also needs approval)

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -78,6 +78,7 @@ require.config({
     'd3-zoom': '../node_modules/d3-zoom/build/d3-zoom',
     'd3-drag': '../node_modules/d3-drag/build/d3-drag',
     'virtual-dom': '../node_modules/virtual-dom/dist/virtual-dom',
```
Comment (genofire) [inline app.js]: delete `virtual-dom` ...

File: lib/proportions.js [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
     'use strict';
 
     return function (config, filterManager) {
       var self = this;
       var scale = d3Interpolate.interpolate('#770038', '#dc0067');
+      V = V.default;
 
-      var statusTable = document.createElement('table');
-      statusTable.classList.add('proportion');
-      var fwTable = statusTable.cloneNode(false);
```
Comment (genofire) [inline lib/proportions.js:11]: :+1:

File: lib/sorttable.js [hunk 3]
```diff
@@ -1,11 +1,12 @@
-define(['virtual-dom'], function (V) {
+define(['snabbdom'], function (V) {
   'use strict';
+  V = V.default;
 
   return function (headings, sortIndex, renderRow) {
+    var self = this;
     var data;
     var sortReverse = false;
     var el = document.createElement('table');
```
Comment (genofire) [inline lib/sorttable.js]: has el existis at his own? (could it not be always self.el)?

**Review by genofire — CHANGES_REQUESTED** [review]

File: lib/sorttable.js (hunk 3 above)
Comment (xf-) [reply lib/sorttable.js]: yes, patch needs a element to patch. i can try to use null

Comment (genofire) [reply lib/sorttable.js]: i mean, you could use: `self.el = document.createElement('table')`

File: lib/sorttable.js [hunk 4]
```diff
@@ -55,17 +56,17 @@ define(['virtual-dom'], function (V) {
       }
 
       var elNew = V.h('table', children);
-      el = V.patch(el, V.diff(elLast, elNew));
-      elLast = elNew;
+      el = V.patch(el, elNew);
```
Comment (genofire) [inline lib/sorttable.js]: and merge this two lines two to one

**Pull request merged**